### DISCUSSION TITLE
Pin Channels

### DISCUSSION THREAD

**Discussion by tarunkhatri** [GetStream/stream-chat-android#4574] [Ideas]
We are developing SAAS product in which 1 customer might be chatting with multiple store users thats why we need **Pin Channel** feature. I am not able to find this feature as of now, are you guys thinking to develop this in near future?

**Comment by tarunkhatri** [comment]
We  purchased the startup plan on the assurance of the sales team that this feature is in works and will be incorporated in the next few months.


Please let us know  it this is in works or for now its in the backlog ?

**Comment by andremion** [comment]
Channel pin is available in the SDK since the `6.8.0` release
